ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml

```diff
@@ -0,0 +1,123 @@
+base_images:
+  cli:
+    name: "4.8"
+    namespace: ocp
+    tag: cli
+  opm-builder:
+    name: opm-builder
+    namespace: ci
+    tag: v1.14.3
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "7"
+  os-min8:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make build
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+images:
+- dockerfile_path: openshift-ci/Dockerfile.deploy
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift-kni/performance-addon-operators/build/_output/bin/performance-addon-operators
+  to: performance-addon-operator
+- dockerfile_path: openshift-ci/Dockerfile.must-gather
+  from: os-min8
+  to: performance-addon-operator-must-gather
+- dockerfile_path: openshift-ci/Dockerfile.index-gen.ci
+  from: opm-builder
+  to: performance-addon-operator-index-gen
+- dockerfile_path: openshift-ci/Dockerfile.index.ci
+  from: opm-builder
+  inputs:
+    performance-addon-operator-index-gen:
+      paths:
+      - destination_dir: .
+        source_path: /database/index.db
+  to: performance-addon-operator-index
+- dockerfile_path: openshift-ci/Dockerfile.bundle.upstream.dev
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift-kni/performance-addon-operators/build
+  to: performance-addon-operator-bundle
+- dockerfile_path: openshift-ci/Dockerfile.src.ci
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: pao-src-oc
+operator:
+  bundles:
+  - dockerfile_path: openshift-ci/Dockerfile.bundle.ci
+  - dockerfile_path: openshift-ci/Dockerfile.bundle-4.7.ci
```

Comment:
is openshift-ci/Dockerfile.bundle-4.7.ci  the right docker bundle file ?  (not 4.8?)

Reply:
yep, it for the upgrade test
so 
`openshift-ci/Dockerfile.bundle.ci` - build 4.8 bundle
`openshift-ci/Dockerfile.bundle-4.7.ci` - build 4.7 bundle